.github/workflows/ci.yml

```diff
@@ -43,6 +43,16 @@ jobs:
         java:
           - '8'
           - '17'
+        exclude:
+          - type: 'JS'
+            scala: '2.13.16'
+        include:
+          - type: 'JS'
+            scala: '2.13.15'
+            java: 17
+          - type: 'JS'
+            scala: '2.13.15'
+            java: 8
```

Comment:
we could also do (if i read the [doc](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#expanding-or-adding-matrix-configurations) correctly):
```
matrix:
  type: // no change
    ...
  scala: // remove 2.13
    - '2.12.20'
    - '3.3.4'
  java: // no change
    ...
  include:
    - scala '2.13.16'
      type: [ 'JVM', 'Native' ] 
    - scala '2.13.15'
      type: 'JS'
```

Reply:
Doesn't work unfortunately, you need to specify everything and can't use an array, so this will be longer that now